tests/integration-tests/RtpsDurableReplay/RtpsDurableReplay.mpc

```diff
@@ -0,0 +1,21 @@
+project(publisher) : dcpsexe, dcps_test, dcps_cm, dcps_transports_for_test {
```

Comment:
project names need to be unique across the workspace, pick something more specific (see common usage pattern)